app/src/org/commcare/activities/connect/ConnectIdPhoneActivity.java

```diff
@@ -61,25 +72,55 @@ protected void onCreate(Bundle savedInstanceState) {
 
         uiController.setTitle(title);
         uiController.setMessage(message);
+        displayNumber(existing);
+
+    }
 
+    void displayNumber(String fullNumber){
         int code = PhoneNumberHelper.getCountryCode(this);
-        if (existing != null && existing.length() > 0) {
-            code = PhoneNumberHelper.getCountryCode(this, existing);
+        if (fullNumber != null && fullNumber.length() > 0) {
+            code = PhoneNumberHelper.getCountryCode(this, fullNumber);
         }
 
         String codeText = "";
         if (code > 0) {
             codeText = String.format(Locale.getDefault(), "%d", code);
         }
 
-        if (existing != null && existing.startsWith("+" + codeText)) {
-            existing = existing.substring(codeText.length() + 1);
+        if (fullNumber != null && fullNumber.startsWith("+" + codeText)) {
+            fullNumber = fullNumber.substring(codeText.length() + 1);
         }
 
-        uiController.setPhoneNumber(existing);
+        uiController.setPhoneNumber(fullNumber);
         uiController.setCountryCode(codeText);
     }
 
+
+    private void showPhoneNumberPickerDialog() {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                SubscriptionManager  subscriptionManager = (SubscriptionManager) getSystemService(Context.TELEPHONY_SUBSCRIPTION_SERVICE);
+
+                List<SubscriptionInfo> subscriptionInfoList = subscriptionManager.getActiveSubscriptionInfoList();
+
+
+            if (subscriptionInfoList != null && !subscriptionInfoList.isEmpty()) {
```

Comment:
Nit: indentation is off here, this if should be indented the same as the line above.